plugins/android_boot/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Bootloaders
+
+## Introduction
+
+This plugin is used to update hardware that use partitions to store their firmware on.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob as a Raw Disk File
+in the IMG format. The firmware blob will be flashed to the partition.
+
+This plugin supports the following protocol ID:
+
+* be.dylanvanassche.android_boot
```

Comment:
Is `be.dylanvanassche` appropriate? Not that I have any better ideas, but using your name for essentially a dd seems a little over the top :)